.github/ISSUE_TEMPLATE/support.md

```diff
@@ -1,6 +1,6 @@
 ---
 name: Support
-about: If you have questions about the Prometheus Operator. If you have questions about helm installation, go to https://github.com/prometheus-community/helm-charts repository. If you have questions about kube-prometheus setup, go to https://github.com/prometheus-operator/kube-prometheus
+about: If you have questions about the Prometheus Operator. Helm? Go to https://github.com/prometheus-community/helm-charts. kube-prometheus ? Go to https://github.com/prometheus-operator/kube-prometheus
```

Comment:
```suggestion
about: For questions about prometheus-operator. For helm, go to https://github.com/prometheus-community/helm-charts. For kube-prometheus, go to https://github.com/prometheus-operator/kube-prometheus.
```

Reply:
(nit) I'm good either way

Reply:
I like the suggestion just because it reads better but I'm also fine wither way

Reply:
Updated the suggestion :)